app/javascript/spotlight/admin/crop.js

```diff
@@ -257,6 +271,7 @@ export default class Crop {
     var url = this.fileInput.data('endpoint')
     // Every post creates a new image/masthead.
     // Because they create IIIF urls which are heavily cached.
+    var data = this.getData();
```

Comment:
I think this version of data is unused. I think it is fetched in the ajax call below.

Reply:
Thanks. Will remove.

Reply:
This has been removed now.